### PR TITLE
💄 Redesign design tab phase one

### PR DESCRIPTION
**Pull request by EvaMarco**
This PR solves https://tree.taiga.io/project/penpot/us/4982?milestone=356300

### CODE REVIEW

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layer.cljs [hunk 1]
```diff
@@ -142,45 +145,89 @@
...
+           :placeholder (tr "settings.multiple")
+           :on-change handle-opacity-change
+           :min 0
+           :max 100
+           :className (stl/css :numeric-input)}]]
+
+
+        [:div {:class (stl/css :actions)}
+         (cond
+           (or (= :multiple (:hidden values)) (not (:hidden values)))
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layer.cljs]: If you use more than two times the same attribute lookup, prefer extracting it to a let binding.
Applies to  `(:hidden values)` and also to `(:blocked values)`

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 2]
```diff
@@ -16,12 +17,48 @@
...
+(defn dir-icons-refactor
+  [val]
+  (case val
+    :row            i/grid-row-refactor
+    :row-reverse    i/row-reverse-refactor
+    :column         i/column-refactor
+    :column-reverse i/column-reverse-refactor))
+
+(mf/defc direction-btn
+  [{:keys [dir saved-dir set-direction icon?] :as props}]
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs]: I suggest you to change `set-direction` to `on-change`.
Prefer using standard callback naming convention, that makes clear visual distinction which attrs are functions and which ones are normal values.

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 3]
```diff
@@ -120,132 +157,315 @@
...
+       [:& radio-button {:value "end"
+                         :icon  (get-layout-flex-icon-refactor :align-items :end is-col?)
+                         :title "Align items end"
+                         :id    "align-items-end"}]]
+
+      [:div.align-items-style
+        [:button.align-start.tooltip.tooltip-bottom
+        {:class    (dom/classnames :active  (= align-items :start))
+         :alt      "Align items start"
+         :on-click #(set-align :start)}
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs]: Please don't pass a newly created callbacks (not memoized ones), prefer use the `:data-align "start"` approach

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 4]
```diff
@@ -120,132 +157,315 @@
...
+       [:& radio-button {:value      "space-evenly"
+                         :icon       (get-layout-flex-icon-refactor :align-content :space-evenly is-col?)
+                         :title      "Align content space-evenly"
+                         :id         "align-content-space-evenly"}]]
+      [:*
+       [:div.align-content-style
+        [:button.align-content.tooltip.tooltip-bottom
+         {:class    (dom/classnames :active  (= align-content :start))
+          :alt      "Align content start"
+          :on-click #(set-align-content :start)}
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs]: The same here

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 5]
```diff
@@ -335,46 +646,139 @@
...
+      [:div {:class (stl/css :gap-group)}
+       [:div {:class (stl/css-case :row-gap true
+                                   :disabled (and (= :nowrap wrap-type) (not is-col?)))
+              :title "Row gap"}
+        [:span {:class (stl/css :icon)}
+         i/gap-vertical-refactor]
+        [:> numeric-input* {:className (stl/css :numeric-input true)
+                            :no-validate true
+                            :placeholder "--"
+                            :on-focus (fn [event]
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs:676]: The same here, you pass callbacks as is, which will make the component rerender on all rerenders. This callbacks should be statically defined with use-fn and passed here.

Comment (EvaMarco) [reply frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs:676]: This component should be updated  itself

**Pull request merged**